### PR TITLE
Sdk updates

### PR DESCRIPTION
**Pull request by mfleader**
## Changes introduced with this PR

- [X] Upgrade `arcaflow_plugin_sdk` to make use of changes to `OneOfStringType` and `OneOfStringSchema`
- [X] Update plugin's `ProfileTransaction` schema to use `discriminator_inlined=True`

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Review by jaredoconnell — APPROVED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -9,7 +9,7 @@ packages = [{include = "uperf_plugin.py", from="./arcaflow_plugin_uperf"}]
 
 [tool.poetry.dependencies]
 python = "^3.9"
-arcaflow-plugin-sdk = "~0.13.0"
+arcaflow-plugin-sdk = "~0.14.0.dev1"
```
Comment (webbnh) [inline pyproject.toml]: Is this the correct reference??  Shouldn't it be `0.14.0.b1`?

**Review by webbnh — CHANGES_REQUESTED** [review]
Is that the correct plugin?  (I _think_ that's the one I was talking about wanting to _delete_, yesterday....)

File: pyproject.toml (hunk 1 above)
Comment (webbnh) [reply pyproject.toml]: ~So, this should now be `0.14.1`!~  😁  Nope...wrong repo.

**Review by webbnh — APPROVED** [review]
👍

**Review by webbnh — APPROVED** [review]
🚂

**Review by jaredoconnell — APPROVED** [review]

**Pull request merged**